grakn-dist/src/bin/grakn-redis.sh

```diff
@@ -28,20 +28,46 @@ redisRunning()
     echo $(ps -ef | grep "redis-server" | grep -v grep | awk '{ print $2}')
 }
 
+executeRedisServer(){
+    if [ "$(uname)" == "Darwin" ]; then
+        "${GRAKN_HOME}/bin/"redis-server-osx $1 &
+    elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+        "${GRAKN_HOME}/bin/"redis-server-linux $1 &
```

Comment:
Instead of putting the process into the background, we should use `daemonize yes` in the config.`
That will also create a PID file in /var/run/redis.pid that we can use for the status command.